py-console/xkdb.py

```diff
@@ -242,7 +242,11 @@ def main():
                         help='optionally specify a backend board to connect to')
     args = parser.parse_args()
 
-    backend_servers = get_backend_servers(backend_class=args.type)
+    backend_type = args.type
+    if os.environ.get('CS_CLASS'):
```

Comment:
I would change this to: `if 'CS_CLASS' in os.environ:`
That way you are explicitly checking if it's contained in the environ instead of relying in implicit behavior of get and bool conversion.

Reply:
fixed